src/hardware/vga_crtc.cpp

```diff
@@ -178,14 +178,55 @@ void vga_write_p3d5(io_port_t, io_val_t value, io_width_t)
 		// 		Register (3C0h index 13h).
 		break;
 
-	case 0x09:
+	case 0x09: {
 		if (IS_VGA_ARCH) {
 			vga.config.line_compare = (vga.config.line_compare & 0x5ff) |
 			                          (val & 0x40) << 3;
 		}
+
+		const auto old_val = vga.crtc.maximum_scan_line;
+		const auto new_val = MaximumScanLineRegister{val};
+
 		vga.crtc.maximum_scan_line.data = val;
-		VGA_StartResize();
-		break;
+
+		if (vga.draw.is_double_scanning) {
+			auto flipped = new_val;
+			flipped.start_vertical_blanking_bit9.flip();
+
+			// Only true if only `start_vertical_blanking_bit9` has
+			// been changed and all other bits are unchanged.
+			if (old_val.data == flipped.data) {
```

Comment:
This check was previously expressed as `if ((vga.crtc.maximum_scan_line ^ val) & 0x20)`. I think my way of doing the check is way more readable, but open to suggestions (suggestions that work _with_ `bit_view`, and not against it 😅).

Reply:
Very nice! After reading your comment, the code makes perfect sense, and is self documenting by flipping the vertical blank bit by name 🏅